deploy/sdk/src/dynamo/sdk/cli/build.py

```diff
@@ -602,6 +607,47 @@ def build(
         raise
 
 
+def build_package(
```

Comment:
Can `dynamo build` utilize this as well? Right now `dynamo containerize` is the only command using this function. Also, I feel like all the work in `dynamo build --containerize` should be the same as the work in `dynamo containerize`? They could use a lot more shared code